notifications/notification.go

```diff
@@ -0,0 +1,78 @@
+package notifications
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/b0bbywan/go-mpd-discplayer/config"
+)
+
+const (
+	EventAdd    = "add"
+	EventRemove = "remove"
+	EventError  = "error"
+)
+
+var soundPaths = map[string]string{
+	EventAdd:    filepath.Join(config.SoundsLocation, "in.mp3"),
+	EventRemove: filepath.Join(config.SoundsLocation, "out.mp3"),
+	EventError:  filepath.Join(config.SoundsLocation, "error.mp3"),
+}
+
+type Notifier struct {
+	Player
+}
+
+type Player interface {
+	Play(name string) error
+	Close()
+}
+
+// NewRootNotifier creates a new instance of RootNotifier.
+func NewNotifier() *Notifier {
+	var player Player
```

Comment:
sc, err := NewSoundCache(soundsPath)
	if err != nil {
		return nil, fmt.Errorf("Failed to load sound cache: %w", err)
	}